src/components/JobWizard/JobWizardSteps/JobWizardFunctionSelection/JobWizardFunctionSelection.js

```diff
@@ -353,8 +358,48 @@ const JobWizardFunctionSelection = ({
     return openConfirmPopUp('All changes will be lost', confirmHandler)
   }
 
+  const setSelectedCard = index => {
+    selectedCardRef.current = index
+    selectedTempState.current = activeTab
+  }
+
+  useEffect(() => {
+    if (stepIsActive === true) setActiveTab(selectedTempState.current)
+  }, [stepIsActive, setActiveTab, selectedTempState])
+
+  const setDivClass = useCallback(() => {
+    const parentElement = containerRef.current.parentNode
+    parentElement.classList.add('wizard-form-scroll-solution')
+  }, [containerRef])
```

Comment:
add a class name for all div elements in the Wizard component and delete this logic here